tests/conftest.py

```diff
@@ -899,8 +907,6 @@ def tearDownClass(self):
         if not self.keep_shared:
             shutil.rmtree(self.libs)
             shutil.rmtree(self.env)
-        # reset cwd
-        os.chdir(self.cwd)
```

Comment:
You can remove the variable as well I think.

Reply:
Ah right, I hadn't considered that. I think it should be fine, because I can't think of a use case of accessing this field, but you're right that we can't be certain.